.storybook/components/Docs/Guildelines/Tokens.stories.mdx

```diff
@@ -84,133 +87,88 @@ Tier 1 tokens define the brand’s design values. They can be thought of as a br
-- `link-foreground-hover` defines text link hover color
+Defines the border width, radius, and color properties for form controls. These values are all mapped to other Tier 2 values and are considered optional in case remapping is required.
 
-#### Neutral colors
+- `form-border-width` defines the default border width for any form element
+- `form-border-radius` defines the default border radius used on any form element's border
 
-- `neutral-subtle` defines a subtle border
-- `neutral-medium` defines a medium border
-- `neutral-bold` defines a pronounced border
+## Tier 3 Component Tokens <a name="tier-3-component-tokens"></a>
 
-#### Text highlight colors
+Tier 3 component variables represent each component's specific values.
 
-Defines text highlight values
+Naming for tier-3 tokens will use the following prefix:
 
-### Forms <a name="tier-2-forms"></a>
+```css
+--eds-theme-[property]-[component name]
+```
 
-Defines the border width, radius, and color properties for form controls. These values are all mapped to other Tier 2 values and are considered optional in case remapping is required.
+With the rest matching a corresponding use for the given component. For example, if a modal is to have a specific header background color, the token to use is:
 
-### Shadows <a name="tier-2-shadows"></a>
+```css
+--eds-theme-color-brand-header-background
+```
 
-- `box-shadow` defines the shadow value that gets applied by default to buttons and other "elevated" components like cards.
+The suffix here is arbitrary, but uses similar naming to tier two tokens. When possible, any new tier 3 tokens should be a more specifically named tier 2 token, to keep the naming consistent.
 
-## Tier 3 component variables
+See the current set of tier 3 tokens [in storybook](/story/design-tokens-tier-3-component--colors).
 
-Tier 3 component variables represent each component's specific values.
 
 ## Tailwind Class Tokens
 
 If the EDS tailwind config theme is being used, Tier 2 and tier 3 color tokens are available as a part of tailwind utility classes, and can be used to apply to specific attributes to a component. Background(prefix: bg-), border(prefix: border-), and text(prefix: text-) colors will only be available for themselves specifically. e.g.:
 
-```jsx
-<div
-  className="
-    bg-brand-grape-100 // will NOT work since tier 1 colors are not available
-    bg-button-icon-brand // will work since button colors are available across all color utility classes
-    bg-link-neutral // will NOT work since background utility classes don't have border colors
-    border-link-neutral // will work since border colors are available for border color utility classes
-
-    bg-brand-primary-strong
-    border-brand-primary-strong // will reflect respective color utility tokens
-      (background-brand-primary-strong and border-brand-primary-strong)
-  "
->
+### These will not work
```

Comment:
Changed these around b/c the IDE (with the tailwind plugin) was throwing weird warnings. Using HTML layout instead